README.md

```diff
@@ -9,20 +9,20 @@
 
 ## Features
 
-* Amazing human like typing effects beyond your imagination
-* It does typo, and correct them
-* Small size **[1.5k](https://bundlephobia.com/result?p=shabdawali@2.1.0)**
-* No dependency
-* CPU friendly
+* Amazing human-like typing effects beyond your imagination.
+* It finds typos, and corrects them.
+* Small size **[1.5k](https://bundlephobia.com/result?p=shabdawali@2.1.0)**.
+* No dependency.
+* CPU friendly.
 * Dynamic speed and pauses to type/delete sentences.
-* Can be used as jQuery plugin, React component, or with other libraries
-* You can fully control it's behavior
-* You can use it for playing music notes, typing demo, typing tweets, or typing posts on devRant, twitter, github.
-* Check http://amitkumargupta.work for live demo.
+* Can be used as a jQuery plugin, React component, or with other libraries.
+* You can fully control it's behavior.
+* You can use it for playing music notes, typing demos, typing posts on DevRant, Twitter and Github.
+* Check http://amitkumargupta.work for a live demo.
 
 # How to use
 
-You can either install it through npm or use directly js from [dist](dist) folder or from some [CDN](https://unpkg.com/shabdawali@2.0.0/dist/shabdawali.js).
+You can either install it through npm or directly from js in the [dist](dist) folder or from [CDN](https://unpkg.com/shabdawali@2.0.0/dist/shabdawali.js).
```

Comment:
",or download js from dist folder or directly refer to CDN  " seems correct to me.

Reply:
Ok will make necessary corrections